my-app/src/app/toolbar/toolbar.component.css

```diff
@@ -0,0 +1,80 @@
+* {
+    height: 100%;
+    margin: 0;
+}
+
+.nav {
+    border-bottom: 1px solid #EAEAEB;
+    text-align: right;
+    height: 84px;
+    background: rgb(4, 116, 192);
+    background: linear-gradient(90deg, rgba(4, 116, 192, 1) 32%, rgba(17, 118, 196, 1) 43%, rgba(236, 192, 39, 1) 66%);
+}
+
+.logo {
+    width: 19rem;
+    height: 8rem;
+    margin-right: 42rem; 
```

Comment:
If you're using this in order to add space between the logo and the rest of the items in the toolbar, then I would recommend using another div that has a flex-grow of 1 style.